documents/userguide/installation.rst

```diff
@@ -19,6 +19,13 @@ Alternatively, clone the code from `github page <https://github.com/HajimeKawaha
 	  
 	  python setup.py install
 
+If the older version of ExoJAX has already been installed, you need to remove all of the old modules. To do so, the following procedure is recommended:
```

Comment:
```suggestion
If the older version of ExoJAX has already been installed, you need to remove all of the old modules (Otherwise, modules that have been deleted and no more exist in the current version remain). To do so, the following procedure is recommended:
```